platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1572,6 +1605,104 @@ public void startSyncNetwork() {
         }
     }
 
+    /**
+     * Starts sync as a protocol network, with work managed by a {@link NegotiatorThread}
+     * <p>
+     * Starting sync with this method supports emergency reconnect
+     *
+     * @param connectionManagers the constructed connection managers
+     */
+    private void startSyncAsProtocolNetwork(@NonNull final StaticConnectionManagers connectionManagers) {
+        final Duration hangingThreadDuration = platformContext
+                .getConfiguration()
+                .getConfigData(BasicConfig.class)
+                .hangingThreadDuration();
+
+        // if this is a single node network, start dedicated thread to "sync" and create events
+        if (initialAddressBook.getSize() == 1) {
+            syncProtocolThreads.add(new StoppableThreadConfiguration<>(threadManager)
+                    .setPriority(Thread.NORM_PRIORITY)
+                    .setNodeId(selfId.getId())
+                    .setComponent(PLATFORM_THREAD_POOL_NAME)
+                    .setOtherNodeId(selfId.getId())
+                    .setThreadName("SingleNodeNetworkSync")
+                    .setHangingThreadPeriod(hangingThreadDuration)
+                    .setWork(new SingleNodeNetworkSync(
+                            this::checkPlatformStatus,
+                            this::getEventTaskCreator,
+                            this::getSleepAfterSync,
+                            selfId.getId()))
+                    .build(true));
+
+            return;
+        }
+
+        // If we still need an emergency recovery state, we need it via emergency reconnect.
+        // Start the helper now so that it is ready to receive a connection to perform reconnect with when the
+        // protocol is initiated.
+        if (emergencyRecoveryManager.isEmergencyStateRequired()) {
+            reconnectController.get().start();
+        }
+
+        final PeerAgnosticSyncChecks peerAgnosticSyncChecks = new PeerAgnosticSyncChecks(
+                List.of(() -> !gossipHalted.get(), () -> intakeQueue.size() <= settings.getEventIntakeQueueSize()));
```

Comment:
Does spotless not like it if each check in the list is on a separate line? It's much easier to read that way

Reply:
Yes, since it *can* fit on one line, spotless forces it that way